scripts/update-rules.js

```diff
@@ -15,6 +25,29 @@ const tablePlaceholder = /<!--RULES_TABLE_START-->[\S\s]*<!--RULES_TABLE_END-->/
 const EMOJI_STAR = ':white_check_mark:';
 const EMOJI_OCTANE = ':car:';
 const EMOJI_STYLISTIC = ':dress:';
+const EMOJI_FIXABLE = ':wrench:';
+
+function isRuleFixable(ruleName) {
+  const relativePath = ruleName.startsWith('deprecated-')
+    ? `../lib/rules/deprecations/${ruleName}.js`
+    : `../lib/rules/${ruleName}.js`;
+  const pathRule = path.resolve(__dirname, relativePath);
+  let rule = fs.readFileSync(pathRule, { encoding: 'utf8' });
+
+  let ast = parse(rule, { sourceType: 'module' });
+
+  let isFixable = false;
+
+  traverse(ast, {
+    ObjectProperty(path) {
+      if (path.node.key.name === 'isFixable' && path.node.value.value !== false) {
```

Comment:
I would check more types here, especially to avoid crashing it if encounters a `SpreadElement` like `...moreProps`.

```suggestion
      if (
        path.node.key.type === 'Identifier' &&
        path.node.key.name === 'isFixable' &&
        !(path.node.value.type === 'Identifier' && path.node.value.value === false)
      ) {
```